client/components/Comments/CommentsDumb.js

```diff
@@ -160,14 +167,17 @@ class Comments extends Component {
                     </div>
                   </Comment.Metadata>
                   <Comment.Text>{comment.content}</Comment.Text>
-                  <Comment.Actions>
-                    <Comment.Action>
-                      Reply{' '}
-                      {comment.comments.length
-                        ? `(${comment.comments.length})`
-                        : ''}
-                    </Comment.Action>
-                  </Comment.Actions>
+                  { this.props.type === 'post' ?
```

Comment:
Why remove threads for events?

Reply:
We don't have to remove threads for events. 
I actually thought we decided that Event comments were not threaded, and more linear.
I can remove that check on 'post'